### PR TITLE
qt5-build.eclass: Cleanup quirks for no longer available versions

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

**Comment by a17r** [comment]
yeah, that's fine by me as well - so qtwebkit will remain in Gentoo ebuild repo only.

**Comment by Pesa** [comment]
yup

**Comment by a17r** [comment]
want me to merge already or wait for 5.9.6 to leave the main repo?

**Comment by Pesa** [comment]
up to you

**Comment by a17r** [comment]
thanks, done ;)

File: eclass/qt5-build.eclass [hunk 1]
```diff
@@ -535,13 +523,11 @@ qt5_base_configure() {
...
 			fi
 		fi)
-		$(if [[ ${QT5_MINOR_VERSION} -ge 10 ]]; then
-			if use kernel_linux; then
-				if tc-is-gcc; then
-					echo -platform linux-g++
-				elif tc-is-clang; then
-					echo -platform linux-clang
-				fi
+		$(if use kernel_linux; then
```
Comment (Arfrever) [inline eclass/qt5-build.eclass]: The 2 uses of `$( )` here could be merged into one `$( )`:
```
		$(if use kernel_FreeBSD; then
			if tc-is-gcc; then
				echo -platform freebsd-g++
			elif tc-is-clang; then
				echo -platform freebsd-clang
			fi
		elif use kernel_linux; then
			if tc-is-gcc; then
				echo -platform linux-g++
			elif tc-is-clang; then
				echo -platform linux-clang
			fi
		fi)
```

**Pull request closed without merging**